code/modules/research/xenobiology/xenobiology.dm

```diff
@@ -762,6 +762,12 @@
 	if(SM.sentience_type != animal_type)
 		to_chat(user, "<span class='warning'>You cannot transfer your consciousness to [SM].</span>" )
 		return ..()
```

Comment:
this already keeps you from transfering your mind into bots, such as medibots

Reply:
yeah sentience_type handles this already, no need for this